plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -404,7 +404,8 @@ protected boolean isSupportedJoinCondition(ConnectorSession session, JdbcJoinCon
 
     public static LongWriteFunction trinoDateToOracleDateWriteFunction()
     {
-        return new LongWriteFunction() {
```

Comment:
why no checkstyle complains?

Reply:
🤷‍♂️

Reply:
I'll take a look.

Reply:
Looks like "expression blocks" are not subject to RightCurly checks. There's an open issue to add it - https://github.com/checkstyle/checkstyle/issues/5945

For the LeftCurly we need to add https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#OBJBLOCK to the list at https://github.com/airlift/airbase/blob/b718176e97bf2748b90dfb150ed940c08883f2b2/airbase-policy/src/main/resources/checkstyle/airbase-checks.xml#L138-L141

But since there's no way to configure same for RightCurly do you think it'd be useful at all @findepi ?

Reply:
i think left curly is where were err more often, so makes sense to me

Reply:
It works but has a much wider scope. The biggest problem though is that IntelliJ can't be configured to follow it and if you are writing new code it auto-indents things in a way that fail checkstyle.

We can enforce the reverse though - i.e. all expressions use braces at end-of-line (similar to if, for, while etc.). i.e. opposite of the change you made here.

Reply:
let's have this discussion on #dev maybe